lib/charms/layer/jenkins/plugins.py

```diff
@@ -19,21 +17,18 @@ def install(self, plugins):
 
         @params plugins: A whitespace-separated list of plugins to install.
         """
+        hookenv.log("Starting plugins installation process")
         plugins = plugins or ""
         plugins = plugins.split()
-        hookenv.log("Stopping jenkins for plugin update(s)")
-        host.service_stop("jenkins")
-        hookenv.log("Installing plugins (%s)" % " ".join(plugins))
+        plugins = (self._get_plugins_to_install(plugins))
```

Comment:
You have some unnecessary brackets here. Maybe you wanted to cast it to a set or list or make a mutable copy?

Reply:
I was doing something to that but after removing the processing I forgot the brackets.

Reply:
2e5d3b6b01ce6008570eff7efb8690447b0ee9ee